addons/website_sale/models/sale_order.py

```diff
@@ -373,7 +373,14 @@ def _cart_update(self, product_id, line_id=None, add_qty=0, set_qty=0, **kwargs)
             'warning': warning,
         }
 
-    def _cart_find_product_line(self, product_id, line_id=None, **kwargs):
+    def _cart_find_product_line(
+            self,
+            product_id,
+            line_id=None,
+            linked_line_id=False,
+            optional_product_ids=None,
+            **kwargs
```

Comment:
```suggestion
            **kwargs,
```
Pure nitpicking, only if you have to change something else